posthog/api/test/test_session_recordings.py

```diff
@@ -10,375 +11,375 @@
 from posthog.helpers.session_recording import Event, compress_and_chunk_snapshots
 from posthog.models import Organization, Person
 from posthog.models.session_recording_event import SessionRecordingViewed
+from posthog.models.session_recording_event.util import create_session_recording_event
 from posthog.models.team import Team
 from posthog.test.base import APIBaseTest
 
 
-def factory_test_session_recordings_api(session_recording_event_factory):
-    class TestSessionRecordings(APIBaseTest):
-        def create_snapshot(
-            self,
-            distinct_id,
-            session_id,
-            timestamp,
-            team_id=None,
-            window_id="",
-            source=0,
-            has_full_snapshot=True,
-            type=2,
-        ):
-            if team_id is None:
-                team_id = self.team.pk
-            session_recording_event_factory(
-                team_id=team_id,
+def _create_session_recording_event(**kwargs):
```

Comment:
For reviewers  - this file is just an indentation due to removing the old ee folder test